filebeat/testing/integration/deprecated_test.go

```diff
@@ -0,0 +1,104 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/elastic/beats/v7/libbeat/testing/integration"
+)
+
+func TestFilebeatDeprecated(t *testing.T) {
+	ctx, cancel := context.WithTimeout(t.Context(), 5*time.Minute)
+	defer cancel()
+	EnsureCompiled(ctx, t)
+
+	messagePrefix := "sample test message"
+	fileCount := 1
+	lineCount := 1
+
+	reportOptions := integration.ReportOptions{
+		PrintLinesOnFail:  10,
+		PrintConfigOnFail: false,
+	}
+
+	t.Run("check that harvesting works with deprecated input_type", func(t *testing.T) {
+
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+		defer cancel()
+
+		config := `
+filebeat.inputs:
+  - input_type: filestream
```

Comment:
It looks like the deprecated `input_type` does not support `filestream`.

I'm not sure if we even need this test to be honest. `input_type` is no longer documented.

Let's remove this file.